docs/reference/deprecated.md

```diff
@@ -6,6 +6,8 @@ Gutenberg's deprecation policy is intended to support backwards-compatibility fo
 - `wp.components.Draggable` as a DOM node drag handler has been removed. Please, use `wp.components.Draggable` as a wrap component for your DOM node drag handler.
 - `wp.i18n.getI18n` has been removed. Use `__`, `_x`, `_n`, or `_nx` instead.
 - `wp.i18n.dcnpgettext` has been removed. Use `__`, `_x`, `_n`, or `_nx` instead.
+- `wp.blocks.setUnknownTypeHandlerName` has been removed. Please use `setFreeformContentHandlerName` and `setUnregisteredTypeHandlerName` instead.
+- `wp.blocks.getUnknownTypeHandlerName` has been removed. Please use `getFreeformContentHandlerName` and `getUnregisteredTypeHandlerName` instead.
```

Comment:
From [another comment]():
> Both in this file and in `store/*`, can't we simply treat the deprecated `*UnknownType*` functions as aliases for their `*FreeformContent*` counterparts? If I read the diff correctly, one deprecated function keeps calling a deprecated action creator, which keeps outputting an action of deprecated type.

It seems like it might be better to just stick with `setUnknownTypeHandlerName` since the function isn't often used. An "unknown" type might be distinct enough from an "unregistered" type to leave as-is, but I am happy to use an alias if you prefer.

Reply:
I've been away from this PR for too long. I am reminded now that the naming for the store is less clear with `setFallbackBlockName`. Aliasing seems to make more sense because of this case.